enterprise/app/services/captain/llm/contact_notes_service.rb

```diff
@@ -43,6 +44,11 @@ def chat_parameters
     }
   end
 
+  def account_language
+    account_locale = @conversation.account.locale
+    ISO_639.find(account_locale)&.english_name&.downcase || 'english'
```

Comment:
Would this work for pt_BR?

Reply:
<img width="318" alt="Screenshot 2025-03-12 at 4 53 44 PM" src="https://github.com/user-attachments/assets/d394491b-eff9-4ed3-8a1f-b3c4a8240e08" />

Reply:
<img width="378" alt="Screenshot 2025-03-12 at 7 27 01 PM" src="https://github.com/user-attachments/assets/37abe01e-4710-4e3c-b564-eebb558e1257" />

Reply:
We'll have to strip out the country code before sending it. I don't think there's any lib which will handle `pt_BR` directly, I'll check anyway